rdflib/graph.py

```diff
@@ -347,6 +348,9 @@ def __init__(
         self.context_aware = False
         self.formula_aware = False
         self.default_union = False
+        resource.setrlimit(resource.RLIMIT_STACK, (2 ** 29, -1))
+        sys.setrecursionlimit(10 ** 6)
+        
```

Comment:
Why can't users of this library not just set the recursion limit as they desire?